tests/tests_service.yml

```
@@ -168,8 +172,12 @@
                 destination:
                   - 1.1.1.1
                   - 1::1
-                helper_module: ftp
                 protocol: icmp
+                # these two don't exist yet in RHEL 7
+                helper_module: "{{ 'ftp'
+                  if ansible_distribution in ['RedHat', 'CentOS', 'Fedora']
+                  and ansible_distribution_major_version is version('8', '>=')
+                  else [] }}"
```

```suggestion
                  else omit }}"
```
? or does it need to be `[]` in this context?

---

Thanks -- no, that's a typo, accidentally copied from the `include:` bit. That ought to be `omit` as well, but I didn't want to do unrelated changes.